iaso/api/instances/instances.py

```diff
@@ -927,11 +927,8 @@ def import_data(instances, user, app_id):
                 oucr.org_unit = instance.org_unit
                 if user and not user.is_anonymous:
                     oucr.created_by = user
-                previous_reference_instances = list(instance.org_unit.reference_instances.all())
-                new_reference_instances = list(filter(lambda i: i.form != instance.form, previous_reference_instances))
-                new_reference_instances.append(instance)
-                oucr.save()
-                oucr.new_reference_instances.set(new_reference_instances)
```

Comment:
@madewulf Is there a reason you were adding the previous instances to the change request?